debian/control

```diff
@@ -24,13 +24,17 @@ Depends:
  pt-miniscreen,
 # Needed to update the EEPROM
  pt-os-notify-services,
+# To provide access to 'wpa_gui' via http
+ pt-web-vnc,
```

Comment:
should this be in the python deps as well or instead maybe? is used both ways. should we be adding a version nuber

Reply:
right! I think when this PR was created, `pt-web-vnc` didn't include a python module. I'll replace this dependency with one in `setup.cfg`